Sources/SourceKit/SourceKitServer.swift

```diff
@@ -523,3 +570,54 @@ public func languageService(
 
 public typealias Notification = LanguageServerProtocol.Notification
 public typealias Diagnostic = LanguageServerProtocol.Diagnostic
+
+extension IndexSymbolKind {
+  func toLspSymbolKind() -> SymbolKind {
+    switch self {
+    case Self.class:
```

Comment:
Style: prefer `.class` over `Self.class` for enum and OptionSet values.  Also, I would hoist the cases onto a single line since there aren't that many of them.